perllib/FixMyStreet/Cobrand/Brent.pm

```diff
@@ -135,6 +136,101 @@ sub geocoder_munge_results {
+        srsname => "urn:ogc:def:crs:EPSG::27700",
+        typename => "BrentCuts",
+        filter => "<Filter><Contains><PropertyName>Geometry</PropertyName><gml:Point><gml:coordinates>$x,$y</gml:coordinates></gml:Point></Contains></Filter>",
+        outputformat => 'GML3',
+    };
+
+    my $features = $self->_fetch_features($cfg, $x, $y, 1);
+
+    if ($$features[0]) {
+        return $$features[0]->{'ms:BrentCuts'}->{'ms:Council'};
+    }
+}
+
+sub munge_overlapping_asset_bodies {
+    my ($self, $bodies) = @_;
+
+    my $in_area = scalar(%{$self->{c}->stash->{all_areas}}) == 1 && (values %{$self->{c}->stash->{all_areas}})[0]->{id} eq $self->council_area_id->[0];
+
+    if ($in_area && $self->check_report_is_on_cobrand_asset) {
+        %$bodies = map { $_->id => $_ } grep {
+            $_->name ne 'Camden Borough Council' &&
+            $_->name ne 'Barnet Borough Council' &&
+            $_->name ne 'Harrow Borough Council'
+            } values %$bodies;
+    } elsif ($in_area && (my $cobrand = $self->check_report_is_on_non_brent_asset)) {
+        my %cobrands = (Harrow => 'Harrow Borough Council', Camden => 'Camden Borough Council', Barnet => 'Barnet Borough Council');
+        my $selected = $cobrands{$cobrand};
+        %$bodies = map { $_->id => $_ } grep {
+            $_->name eq $selected || $_->name eq 'Brent Council' || $_->name eq 'TfL' || $_->name eq 'National Highways'
+        } values %$bodies;
+    } elsif (!$in_area && !$self->check_report_is_on_cobrand_asset) {
+        %$bodies = map { $_->id => $_ } grep {
+            $_->name ne 'Brent Council'
+            } values %$bodies;
+    }
+}
+
+sub munge_cobrand_asset_categories {
+    my ($self, $contacts) = @_;
+
+    my $in_area = scalar(%{$self->{c}->stash->{all_areas}}) == 1 && (values %{$self->{c}->stash->{all_areas}})[0]->{id} eq $self->council_area_id->[0];
+
+    if (!$in_area && $self->check_report_is_on_cobrand_asset) {
+        @$contacts = grep { !($_->email !~ /^Symology/ && $_->body_id == $self->body->id) } @$contacts;
+    } elsif ($in_area && (my $cobrand = $self->check_report_is_on_non_brent_asset)) {
+        my %ids = (
+            'Barnet' => FixMyStreet::DB->resultset("Body")->search({name => 'Barnet Borough Council'})->first->id,
+            'Camden' => FixMyStreet::DB->resultset("Body")->search({name => 'Camden Borough Council'})->first->id,
+            'Harrow' => FixMyStreet::DB->resultset("Body")->search({name => 'Harrow Borough Council'})->first->id,
+            );
+
+        my %exclude = (
+            'Barnet' => { 'Pavements' => 1, 'Potholes' => 1, 'Street cleaning' => 1 },
+            'Camden' => { 'Pavements' => 1, 'Potholes' => 1 },
+            'Harrow' => { 'Pavements' => 1, 'Potholes' => 1 },
+            );
+
+        @$contacts = grep { !($_->email =~ /^Symology/ && $_->body_id == $self->body->id) } @$contacts;
+        @$contacts = grep { !($exclude{$cobrand}{$_->category} && $_->body_id == $ids{$cobrand})} @$contacts;
```

Comment:
Do you want the reverse here? It's remove Brent "roads" and other "roads" rather than other "non-roads" I think?

Reply:
So Brent roads are Echo? If we exclude Brent roads and other roads, do we not have no roads?

Reply:
Sorry if I wasn't clear. You're removing Brent Symology (roads stuff), okay, but you're also removing Barnet/Camden/Harrow roads, rather than removing their non-roads.